### PR TITLE
fast build: watch & reload head CSS files instead of restarting app when in development

### PR DESCRIPTION
**Pull request by bjfletcher**

### CODE REVIEW

File: src/middleware/assets.js [hunk 1]
```diff
@@ -77,6 +80,20 @@ Or \`rm -rf bower_components/n-ui && bower install n-ui\` if you're no longer wo
 			return currentHeadCsses;
 		}, {}) : {};
 
+	if (process.argv.indexOf('--dev') !== 0) {
```
Comment (wheresrhys) [inline src/middleware/assets.js]: Not sure what is most robust, but process.NODE_ENV !== 'production' is another option.

File: src/middleware/assets.js [hunk 2]
```diff
@@ -77,6 +80,20 @@ Or \`rm -rf bower_components/n-ui && bower install n-ui\` if you're no longer wo
 			return currentHeadCsses;
 		}, {}) : {};
 
+	if (process.NODE_ENV !== 'production') {
+		const paths = Object.keys(headCsses).map(css => `${directory}/public/${css}.css`);
+		chokidar.watch(paths).on('change', (path) => {
+			fs.readFile(path, 'utf-8', (err, content) => {
```
Comment (ironsidevsquincy) [inline src/middleware/assets.js]: could use Matt's great https://github.com/matthew-andrews/denodeify lib to turn this into a promise

File: src/middleware/assets.js [hunk 3]
```diff
@@ -77,6 +80,20 @@ Or \`rm -rf bower_components/n-ui && bower install n-ui\` if you're no longer wo
...
+				if (content) {
```
Comment (ironsidevsquincy) [inline src/middleware/assets.js]: why this check? is deleting the file's content a valid user action

**Comment by bjfletcher** [comment]
@ironsidevsquincy excellent suggestions, thanks muchly :)

If you think this looks good, please go ahead and merge - or let me know.

Cheers 🎉

**Pull request merged**